charts/karpenter/values.yaml

```diff
@@ -134,6 +134,9 @@ logLevel: info
 # -- Cluster name.
 clusterName: ""
 
+# -- Minimum kubernetes version that will be supported
+kubernetesMinVersion: "1.19.0-0"
```

Comment:
should we include this in values.yaml?  I don't think anyone is going to be touching this, ideally we should know what min version is supported with Karpenter and hardcode into deployment.yaml.  values.yaml should be things that a user would need to change.  Maybe the thought is that they test with a lower version of k8s than we have and could configure it here?

Reply:
good catch. Done.